seqr/views/apis/anvil_workspace_api.py

```diff
@@ -155,12 +163,18 @@ def create_project_from_workspace(request, namespace, name):
         project, individual_records=pedigree_records, user=request.user
     )
 
-    # Send an email to all seqr data managers
+    # Upload sample IDs to a file on Google Storage
+    ids_path = '{}base/{guid}_ids.txt'.format(_get_loading_project_path(project, request_json['sampleType']), guid=project.guid)
+    sample_ids = '\n'.join(['s'] + [individual.individual_id for individual in updated_individuals])
     try:
-        _send_load_data_email(project, updated_individuals, data_path, request_json['sampleType'], request.user)
+        temp_path = save_temp_data(sample_ids)
+        mv_file_to_gs(temp_path, ids_path, user=request.user)
     except Exception as ee:
-        message = 'AnVIL loading request email exception: {}'.format(str(ee))
-        logger.error(message, request.user)
+        logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
+                     detail=sample_ids)
```

Comment:
for the detail its better to send the list of ids rather than the string

Reply:
Good to know that `detail` can be a list or any other object.